src/app/test-suites/root-cause-analysis/page.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Root Cause Analysis
+description: Root Cause Analysis identifies the underlying issues in your application responsible for the majority of test failures.
+image: /images/Root-cause-analysis.png
+---
+
+## Overview
+
+End to end tests have a lot of advantages. They are relatively easy to write. And they are the best way to ensure that a change is safe to land. The primary downside is that they fail in surprising ways.
+
+This makes sense. There are lots of reasons why a button might not be visible or clickable. And when the test fails, the error message describes the failed assertion or failed step, but cannot actually tell you what went wrong in the application.
+
+This is where Root Cause Analysis comes in. We programmatically compare passing and failing tests to find th earliest divergences and categorize the failure based on what's happening in the application, not the test.
+
+The result is that instead of showing hundreds of test error messages, we're able to show a couple of the underlying issues in your application that are responsible for the majority of test failures.
+
+### How are failures analyzed?
+
+At replay time we look for earliest point where the failing test diverges. A common example is when a function is called and finishes when the test passes, but does not finish when the test fails. There are two reasons why this could happen: either the function threw an exception or had a promise that did not resolve.
```

Comment:
```suggestion
At replay time we look for the earliest point where the failing test diverges. A common example is when a function is called and finishes when the test passes, but does not finish when the test fails. There are two reasons why this could happen: either the function threw an exception or had a promise that did not resolve.
```